GI03FLMB/GI03FLMB_D00007_ingest.csv

```diff
@@ -0,0 +1,28 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00007/*status_1487701.dat,GI03FLMB-RIM01-00-SIOENG000,telemetered,Available,
+mi.dataset.driver.adcps_jln.sio.adcps_jln_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00007/*adcps*.dat,GI03FLMB-RIM01-02-ADCPSL007,telemetered,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG000,telemetered,Available,Wildcard refdes
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG060,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG061,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG062,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG063,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG064,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG065,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG066,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG067,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOG068,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH069,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH070,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00007/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH071,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00007/*status_139901.dat,GI03FLMB-RIS01-00-SIOENG000,telemetered,Available,

```

Comment:
Do we want the S/N in here instead of a "*"?

Reply:
Should this be 1399001? All other SSOIC numbers are 7 digits – 6 digit S/N with 01 at the end.  This S/N is 13990.

Reply:
Yes it should be 1399001. Good catch! Will push a new version with that fix